src/server/middleware/contenttrust/notaryv1.go

```diff
@@ -28,8 +28,8 @@ var (
 	}
 )
 
-// Middleware handle docker pull content trust check
-func Middleware() func(http.Handler) http.Handler {
+// NotaryV1 handle docker pull content trust check
+func NotaryV1() func(http.Handler) http.Handler {
```

Comment:
is there version 2 of the notary? If not I think it's enough to name it as `Notary` here.

Reply:
yes, there is. https://github.com/notaryproject/notaryproject

Reply:
it's ok to use notary.